reportportal/templates/analyzer-statefulset.yaml

```diff
@@ -58,6 +58,10 @@ spec:
           value: "analyzer-default"
         - name: ES_HOSTS
           value: "{{ .Values.elasticsearch.endpoint }}"
+        {{- if .Values.serviceanalyzer.uwsgiWorkers }}
+        - name: UWSGI_WORKERS
+          value: "{{ .Values.serviceanalyzer.uwsgiWorkers }}"
+        {{- else }}
```

Comment:
Please pay attention you specified `{{- else }}` when there is no "else" condition, you should end this if with `{{- end }}`.

Reply:
I'm inconsiderate